packages/code-snippet-block/src/helpers.ts

```diff
@@ -0,0 +1,14 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export const isPxSuffixMissing = (value: string) => !value.includes('px'); // TODO: remove isPxSuffixMissing - use appendUnit after fix
```

Comment:
Are these Todo still up to date?

Reply:
yes, as there is a bug with onChange for custom padding. Not adding 'px' to value.

Reply:
👍